checks/doks/dobs_pod_owner.go

```diff
@@ -67,6 +67,7 @@ func (p *dobsPodOwner) Run(objects *kube.Objects) ([]checks.Diagnostic, error) {
 		}
 	}
 	for _, pod := range dobsPods {
+		pod := pod
```

Comment:
Maybe I'm wrong, but shouldn't this use of iteration variables be okay given that the loop body does not invoke a goroutine? 🤔

Reply:
We spin up goroutines in checks.Run: https://github.com/digitalocean/clusterlint/blob/master/checks/run_checks.go#L51

Reply:
This is not about goroutines. In the check warning, we use &pod.ObjectMeta. Since we're using a reference, the loop variable must be reinitialized according to https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable

Reply:
Ah right, I forgot about that other gotcha. Thanks for explaining.